commit cd3c520b084f8d209baf938a0f78692f68978b33
Author: Yang Bo <yangbo@linuxdeepin.com>
Date:   Tue Apr 19 13:01:44 2016 +0800

    Adjust swappiness.
    
    Adjust swappiness to use cache as much mas possible.
    
    Signed-off-by: Yang Bo <yangbo@linuxdeepin.com>

diff --git a/mm/vmscan.c b/mm/vmscan.c
index 2aec424..2a2eac8 100644
--- a/mm/vmscan.c
+++ b/mm/vmscan.c
@@ -139,7 +139,7 @@ struct scan_control {
 /*
  * From 0 .. 100.  Higher means more swappy.
  */
-int vm_swappiness = 60;
+int vm_swappiness = 10;
 /*
  * The total number of pages which are beyond the high watermark within all
  * zones.
